VoteStakeRegistry/components/Account/Account.tsx

```diff
@@ -49,19 +50,29 @@ const NotificationsPlugin = () => {
   return null
 }
 
+const DelegatePlugin = () => {
+  return (
+    <div className="col-span-6">
+      <DelegateCard />
+    </div>
+  )
+}
+
 const Account = ({ withHeader = true }: { withHeader?: boolean }) => {
```

Comment:
It's not related to this PR but the `Account` component should not be under `VoterStakeRegistry` namespace